### DISCUSSION TITLE
Where can i set Video Memory Size?

### DISCUSSION THREAD

**Discussion by DocMAX** [ptitSeb/box64#1218] [General]
Where can i set Video Memory Size?

**Comment by DocMAX** [comment]
![grafik](https://github.com/ptitSeb/box64/assets/5351323/eb12b2b1-b73d-45a5-a466-b3013421032d)

**Reply by KreitinnSoftware** [reply]
On Wine Regedit
HKEY_CURRENT_USER/Software/Wine/Direct3D/VideoMemorySize

**Reply by KreitinnSoftware** [reply]
[VideoMemorySize_1024.reg.txt](https://github.com/ptitSeb/box64/files/14076551/VideoMemorySize_1024.reg.txt)

**Reply by KreitinnSoftware** [reply]
just rename this .txt to .reg and apply
